src/dialogs/voice-assistant-setup/voice-assistant-setup-step-success.ts

```diff
@@ -213,8 +214,19 @@ export class HaVoiceAssistantSetupStepSuccess extends LitElement {
     });
   }
 
-  private _testTts() {
-    this._announce("Hello, how can I help you?");
+  private async _testTts() {
+    const [pipeline] = await this._getPipeline();
+
+    if (!pipeline) {
+      return;
+    }
+
+    if (pipeline.language !== this.hass.locale.language) {
+      const result = await getTranslation(null, pipeline.language);
```

Comment:
What happens if we have no translations for the pipeline language?

Reply:
```
throw new Error("Language en is not found in metadata");
```

🙈